backend/src/pong/game/game.controller.ts

```diff
@@ -27,7 +27,7 @@ export class GameController {
 			const newGame: Game = await this.gameService.save(createGameDto);
 			return newGame;
 		} catch (e) {
-			throw new BadRequestException();
+			throw new BadRequestException("this user doesn't exist");
```

Comment:
if you try to play the game with a user that doesn't exist for example:
http://localhost:8080/request-game/0

if you click on request, it will show this message